indico/modules/events/registration/client/js/form/fields/DateInput.jsx

```diff
@@ -98,10 +102,14 @@ DateInputComponent.propTypes = {
     'YYYY',
   ]).isRequired,
   timeFormat: PropTypes.oneOf(['12h', '24h']),
+  min: PropTypes.instanceOf(Date),
+  max: PropTypes.instanceOf(Date),
```

Comment:
Let's use (isoformat) strings here. It'll be pretty much impossible to pass a `Date` to this, since this data comes directly from the (JSON) field setup data, with no javascript-side processing.

Reply:
Ah ok